### DISCUSSION TITLE
Is it correct for Dev builds 4.0.0-6133 to 4.0.0-6135 not to ship with the RSP plug-in?

### DISCUSSION THREAD

**Discussion by dftf-stu** [project64/project64#2366] [General]
If you download the following Dev builds from [here](https://www.pj64-emu.com/nightly-builds):
- Project64-Dev-4.0.0-6133-1522f17
- Project64-Dev-4.0.0-6134-90fefed
- Project64-Dev-4.0.0-6135-98b96a6

None of them come with the "RSP 1.7.dll" or "Project64-RSP.dll" file inside the Plugin > RSP folder,
meaning it is not-possible to start the emulator:

![P64_NoRSPPlugIn](https://github.com/project64/project64/assets/83965343/48cbb447-a969-43e7-b16c-5a3e83e2b16d)

Is this intentional?

Seems a bit-odd to publish versions of the app which obviously cannot be used as a necessary file isn't shipped with it...! 🤔

**Comment by project64** [comment]
it is fixed in Dev-4.0.0-6136-f4459fe, any change to github code will generate a build even if it does not work.

The early changes changed RSP file name, I missed updating the packaging the zip file. There was also changes to RSP code which is why there is multiple builds.

There is a reason why I still have v3 as the stable download
